pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java

```diff
@@ -480,14 +498,22 @@ public void setAutoTopicCreation(
             @ApiResponse(code = 404, message = "Tenant or cluster or namespace doesn't exist") })
     public void removeAutoTopicCreation(@Suspended final AsyncResponse asyncResponse,
                                         @PathParam("tenant") String tenant, @PathParam("namespace") String namespace) {
-        try {
-            validateNamespaceName(tenant, namespace);
-            internalRemoveAutoTopicCreation(asyncResponse);
-        } catch (RestException e) {
-            asyncResponse.resume(e);
-        } catch (Exception e) {
-            asyncResponse.resume(new RestException(e));
-        }
+        validateNamespaceName(tenant, namespace);
+        internalSetAutoTopicCreationAsync(null)
+                .thenAccept(__ -> {
+                    log.info("[{}] Successfully remove autoTopicCreation on namespace {}", clientAppId());
+                    asyncResponse.resume(Response.noContent().build());
```

Comment:
Seems forget to put `namespace`

Reply:
log.info("[{}] Successfully remove autoTopicCreation on namespace {}", clientAppId()) -> 
log.info("[{}] Successfully remove autoTopicCreation on namespace {}", clientAppId(), namespaceName);